week-4/C-more-complex-objects/exercise-3.js

```diff
@@ -32,23 +32,43 @@ var parkAvenueHouse = {
 
 // returns the full name (first name + last name) of the owner of the house
 function getOwnerFullName(house) {
-
+  return (
+    kinningParkHouse.currentOwner.firstName +
+    " " +
+    kinningParkHouse.currentOwner.lastName
+  );
 }
 
 // returns an array of the owners' email addresses of the two houses
 function getEmailAddresses(house1, house2) {
-
+  return (
+    kinningParkHouse.currentOwner.email +

```

Comment:
bug: This should work on house1, and house2 - and you shouldn't have to know what they're going to be in advance!  

The syntax is good, though.